README.md

```diff
@@ -63,34 +63,67 @@ Objectives:
 
-### Stable
+#### Stable
 
 The component is ready for general availability. Bugs and performance problems should be reported and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output are not expected to happen without prior notice, unless under special circumstances.
 
-### Deprecated
+#### Deprecated
 
 The component is planned to be removed in a future version and no further support will be provided. Note that new issues will likely not be worked on. When a component enters "deprecated" mode, it is expected to exist for at least two minor releases. See the component's readme file for more details on when a component will cease to exist.
 
+### Metric stability levels
+
+Receivers that produce metrics should declare a stability level for each metric. A for each metric may not exceed the The status for each metric should be listed in the receiver's `documentation.md` file according to the following definitions:
+
+#### In development
+
+The metric may be documented but not available, or available but subject to change at any time. Bugs and performance issues should be reported, but it is likely that the component owners might not give them much attention. Your feedback is still desired. The metric should not be used in production.
+
+#### Alpha
+
+The component is ready to be used for limited non-critical workloads and the authors of this component would welcome your feedback. Bugs and performance problems should be reported, but component owners might not work on them right away. The configuration options might change often without backwards compatibility guarantees.
+
+#### Beta
+
+Same as Alpha, but the configuration options are deemed stable. While there might be breaking changes between releases, component owners should try to minimize them. A component at this stage is expected to have had exposure to non-critical production workloads already during its **Alpha** phase, making it suitable for broader usage.
+
+#### Stable
+
+The component is ready for general availability. Bugs and performance problems should be reported and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output are not expected to happen without prior notice, unless under special circumstances.
+
+#### Deprecated
+
+The component is planned to be removed in a future version and no further support will be provided. Note that new issues will likely not be worked on. When a component enters "deprecated" mode, it is expected to exist for at least two minor releases. See the component's readme file for more details on when a component will cease to exist.
+
+### Stability level progression
```

Comment:
Are you allowed to remove a deprecated metric, or disable to save cost (backend cost as well as runtime cost)?

Reply:
Thanks for the reminder to incorporate default enabled state as well. I will add considerations for this.

Reply:
I've proposed some guidelines for enabled status within each stability level.